www/src/components/account/Domains.js

```diff
@@ -198,20 +207,25 @@ function Domain({ node, last, setDomain }) {
   )
 }
 
-export function Domains() {
+function DomainsInner({ q, setDomainSelected }) {
   const [listRef, setListRef] = useState(null)
   const [domain, setDomain] = useState(null)
   const { data, loading, fetchMore } = useQuery(DNS_DOMAINS, {
```

Comment:
We should modify the api to allow for a search filter and not do it in the frontend (it's ok for now though)

Reply:
Totally agree. This is just a quick fix since it is really useful to be able to filter at all. Will create a follow-up issue to add support in the API and update the frontend.

Reply:
Cool sounds good